packages/mds-logger/index.ts

```diff
@@ -23,7 +24,9 @@ if (inspect?.defaultOptions) {
   inspect.defaultOptions.depth = null
 }
 
-const logger: Pick<Console, 'info' | 'warn' | 'error' | 'debug'> = console
+const destination = pino.destination({ sync: false })
+
+const logger: Pick<Logger, 'info' | 'warn' | 'error' | 'debug'> = pino(destination)
```

Comment:
Seems like these two lines could be combined since `destination` isn't used anywhere else.

Reply:
Yeah, could definitely do that.